### ISSUE TITLE
Keyboard focus not visible after loading Outlook add-in on Outlook Desktop app in Windows

### ISSUE THREAD

**Issue by ctruong9** [OfficeDev/office-js#2397]
# Provide required information needed to triage your issue
After using keyboard to navigate to a link/button to open an Outlook add-in, the focus disappears. After pressing tab an indefinite amount of times, the keyboard focus does not reappear. The only workaround I could find was pressing "space" once, then the immediate next "tab" press will show keyboard focus within the Outlook add-in loaded.

## Your Environment

* Platform [PC desktop, Mac, iOS, Office on the web]: Windows
* Host [Excel, Word, PowerPoint, etc.]: Outlook
* Office version number: 2112 (Build 14729.20260 Click-to-Run)
* Operating System: Windows 10
* Browser (if using Office on the web): N/A
 
## Expected behavior

After using keyboard navigation to go to a link/button to open an Outlook add-in, the keyboard focus should shift into the Outlook add-in automatically.

## Current behavior
Currently, after using keyboard to navigate to a link/button to open an Outlook add-in, the focus disappears after pressing that link/button. Pressing "tab" to find or move focus does not do anything, the only way to regain focus is to press "space" and then "tab" immediately next and that puts the keyboard focus within the Outlook add-in.

## Steps to reproduce

Steps to repro:
1. Open Outlook Desktop App in Windows
2. Ensure you have any Outlook add-in to load
3. Click on an email in the inbox
4. Using your keyboard to navigate with tab or shift+tab, go to an Outlook add-in link/button
5. Press space or enter to activate and load the add-in
6. Notice the keyboard focus disappears
7. From here, the only workaround is mentioned above where you must press "space" then "tab" and the focus appears within the Outlook add-in

## Link to live example(s)

1. ______
2. ______
3. ______

# Provide additional details

1. ______
2. ______
3. ______

## Context
This is causing Accessibility issues in which users who rely on keyboard navigation to go around the Outlook app cannot depend on a reliable way to find keyboard focus after loading an Outlook add-in.

## Useful logs

- [ ] Console errors
- [ ] Screenshots
- [ ] Test file (if only happens on a particular file)

Thank you for taking the time to report an issue. Our triage team will respond to you in less than 72 hours. Normally, response time is <10 hours Monday through Friday. We do not triage on weekends.

**Comment by exextoc** [comment]
Do you have a video you can share so that we can better understand this issue?

**Comment by ctruong9** [comment]
As you can see in the recording provided, I click on "Search People" text input to begin keyboard focus. From there, I use keyboard navigation to go to OneNote add-in button. I activate OneNote add-in pressing "space bar" and from there you can see keyboard focus disappears. While there is no keyboard focus, I am continually pressing tab and you can see from about 10 second to 14 second mark, there is no keyboard focus. At this point, the only workaround I found is to press "space bar" and then "tab". At around the 14 second mark, I do that and you can see the keyboard focus shows back up within the OneNote add-in. The expected behavior should be that the keyboard focus goes immediately into the add-in from open.

https://user-images.githubusercontent.com/35610769/152623172-cd03388b-ce9b-4e42-88f1-e539ed62397f.mov

**Comment by exextoc** [comment]
Thank you for the video. We have fixed the bug and the fix should be available in build 16.0.14816.10000 or later. When an updated build is available depends on the release channel a customer is on.
